luci-app-jd-dailybonus未完全同步上游代码，导致编译失败

```
install -d -m0755 /home/xw/openwrt/lede/build_dir/target-x86_64_musl/luci-app-jd-dailybonus-1.0.0/.pkgdir/luci-app-jd-dailybonus/usr/lib/lua/luci/i18n
po2lmo ./po/zh-cn/jd-dailybonus.po /home/xw/openwrt/lede/build_dir/target-x86_64_musl/luci-app-jd-dailybonus-1.0.0/.pkgdir/luci-app-jd-dailybonus/usr/lib/lua/luci/i18n/jd-dailybonus.zh-cn.lmo
Usage: po2lmo input.po output.lmo
```
上游已经取消了语言文件并更改了依赖，这边还没完全同步过来（

Makefile:70: recipe for target '/home/water/lede/build_dir/target-x86_64_musl/luci-app-jd-dailybonus-1.0.0/.pkgdir/luci-app-jd-dailybonus.installed' failed
make[3]: *** [/home/water/lede/build_dir/target-x86_64_musl/luci-app-jd-dailybonus-1.0.0/.pkgdir/luci-app-jd-dailybonus.installed] Error 1
make[3]: Leaving directory '/home/water/lede/package/lean/luci-app-jd-dailybonus'
time: package/lean/luci-app-jd-dailybonus/compile#0.17#0.10#0.66
package/Makefile:111: recipe for target 'package/lean/luci-app-jd-dailybonus/compile' failed
make[2]: *** [package/lean/luci-app-jd-dailybonus/compile] Error 2
make[2]: Leaving directory '/home/water/lede'
package/Makefile:107: recipe for target '/home/water/lede/staging_dir/target-x86_64_musl/stamp/.package_compile' failed
make[1]: *** [/home/water/lede/staging_dir/target-x86_64_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/water/lede'
/home/water/lede/include/toplevel.mk:222: recipe for target 'world' failed
make: *** [world] Error 2

--- a/package/lean/luci-app-jd-dailybonus/Makefile
+++ b/package/lean/luci-app-jd-dailybonus/Makefile
@@ -8,7 +8,7 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=luci-app-jd-dailybonus
 LUCI_PKGARCH:=all
 PKG_VERSION:=1.0.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -18,7 +18,7 @@ define Package/luci-app-jd-dailybonus
 	SUBMENU:=3. Applications
 	TITLE:=Luci for JD dailybonus Script 
 	PKGARCH:=all
-	DEPENDS:=+node +wget
+	DEPENDS:=+node +wget +lua-cjson +lua-md5 +luasec
 endef
 
 define Build/Prepare
@@ -44,6 +44,7 @@ define Package/luci-app-jd-dailybonus/install
 	$(INSTALL_DIR) $(1)/usr/share/jd-dailybonus
 	$(INSTALL_BIN) ./root/usr/share/jd-dailybonus/*.sh $(1)/usr/share/jd-dailybonus/
 	$(INSTALL_DATA) ./root/usr/share/jd-dailybonus/*.js $(1)/usr/share/jd-dailybonus/
+	$(INSTALL_DATA) ./root/usr/share/jd-dailybonus/*.lua $(1)/usr/share/jd-dailybonus/
 
 	$(INSTALL_DIR) $(1)/usr/share/rpcd/acl.d
 	$(INSTALL_DATA) ./root/usr/share/rpcd/acl.d/* $(1)/usr/share/rpcd/acl.d
@@ -62,9 +63,6 @@ define Package/luci-app-jd-dailybonus/install
 
 	$(INSTALL_DIR) $(1)/usr/lib/lua/luci/view/jd-dailybonus
 	$(INSTALL_DATA) ./luasrc/view/jd-dailybonus/* $(1)/usr/lib/lua/luci/view/jd-dailybonus/
-
-	$(INSTALL_DIR) $(1)/usr/lib/lua/luci/i18n
-	po2lmo ./po/zh-cn/jd-dailybonus.po $(1)/usr/lib/lua/luci/i18n/jd-dailybonus.zh-cn.lmo
 endef
 
 $(eval $(call BuildPackage,luci-app-jd-dailybonus))

